src/Microsoft.ComponentDetection.Common/EnvironmentVariableService.cs

```diff
@@ -17,5 +17,17 @@ public bool DoesEnvironmentVariableExist(string name)
                 .OfType<string>()
                 .FirstOrDefault(x => string.Compare(x, name, true) == 0) != null;
         }
+
+        public string GetEnvironmentVariable(string name)
```

Comment:
Can `DoesEnvironmentVariableExist` be simplied to be

```csharp
return GetEnvironmentVariable(name) != null
```

Reply:
Changed to above.